HLTrigger/Configuration/scripts/hltDumpStream

```diff
@@ -292,7 +291,7 @@ def getStreamsInfoFromProcess(process, prescaleValues, numberOfPrescaleColumns):
     # (in Run-2 HLT menus, smart-PSs are applied to standard Paths in the stream EndPath)
     # (in Run-3 HLT menus, prescales at stream level have not been used yet, plus they cannot be applied inside FinalPaths)
```

Comment:
@cms-sw/hlt-l2 I wasn't sure what to do for this comment. I suppose the `FinalPaths` refers to the FinalPath being removed in this PR.

Reply:
> I wasn't sure what to do for this comment. I suppose the FinalPaths refers to the FinalPath being removed in this PR.

yes, this is also what Marino was referring to in https://github.com/cms-sw/hlt-confdb/issues/101 

> Going back to EndPaths would also give the option to re-introduce per-stream prescale modules (something that was not considered necessary for Run 3, and is not possible to do with FinalPaths).

I would go ahead and remove the comment. Thank you.

Reply:
Yes

Reply:
Thanks! Done.